indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -104,6 +105,10 @@ public final static InputRow parseInputRow(Object value, InputRowParser parser)
     if(parser instanceof StringInputRowParser && value instanceof Text) {
       //Note: This is to ensure backward compatibility with 0.7.0 and before
       return ((StringInputRowParser) parser).parse(value.toString());
+    } else if(parser instanceof StringInputRowParser && value instanceof BytesWritable) {
```

Comment:
You don't think it's worth having something in Druid core that knows how to read text that has been written as BytesWritable? This came up for making Druid work better with Secor, which does this if you ask it to write SequenceFiles: https://github.com/pinterest/secor. With this patch it's possible to read those just by setting your inputFormat to org.apache.hadoop.mapreduce.lib.input.SequenceFileInputFormat and using a normal string parser (assuming the data written by Secor from Kafka was utf-8 encoded and something that Druid understands).

I'm open to deciding that it only makes sense to handle Text without users writing custom parsers, but would like to understand the reason for not working out of the box with the other builtin hadoop string-ish type.